app/services/unpublish_service.rb

```diff
@@ -100,8 +100,8 @@ def check_unpublishable(edition)
     end
   end
 
-  def format_govspeak(text)
-    GovspeakDocument.new(text).to_html
+  def format_govspeak(edition, text)
```

Comment:
minor, but flipping the order of these variables around to be "text, edition" would make it a slightly nicer read.

Reply:
Sure - done.